HW_1/task_1.py

```diff
@@ -0,0 +1,10 @@
+    # Задача 2: Найдите сумму цифр трехзначного числа.
+
+Number = str(input('Введите трехзначное число '))
+if len(Number) != 3:
+    print("Вы ввели не трехзначное число!")
+else:
+    sum = 0
+    for dig in range(len(Number)):
+        sum += int(Number[dig])
```

Comment:
Хорошая идея использовать тут цикл перебора. Да, это решение будет медленнее, чем математическое, но любой программист к нему придет раньше.
Однако, должен заметить, что функция `range` здесь использовать совершенно необязательно. Можно напрямую проходиться по элементам строки, которые и будут цифрами.
```suggestion
    for dig in Number:
        sum += int(dig)
```